src/azure-cli-testsdk/HISTORY.rst

```diff
@@ -3,3 +3,6 @@
 Release History
 ===============
 
+0.1.0+dev
```

Comment:
This package should follow the same scheme.

Reply:
Okay done.